### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sul-devops-team**

### CODE REVIEW

File: spec/models/book_spec.rb [hunk 1]
```diff
@@ -154,12 +143,6 @@
       create_list(:book, 2)
     end
 
-    describe '.count_by_status' do
```
Comment (ndushay) [inline spec/models/book_spec.rb]: is it okay to simply remove the specs instead of fixing them?

Comment (mjgiarlo) [reply spec/models/book_spec.rb]: Removed because they are dupes, I believe? @aaron-collier

Comment (aaron-collier) [reply spec/models/book_spec.rb]: It looked like a line for line duplicate to me...

Comment (aaron-collier) [reply spec/models/book_spec.rb]: That said, I'm happy to fix the spec, but it seems like perhaps that should be a different PR than a dependency update.

**Pull request merged**